samples/apps/copilot-chat-app/README.md

```diff
@@ -69,6 +69,12 @@ and these components are functional:
               option is appropriate for your instance.    
             * You will insert your Azure endpoint key during build of the backend
               API Server
+         
+         * If you are using speech to text as input option then under the `AzureSpeechConfig` block, make sure the following configuration matches your instance:
+            * `"Region": "westus2",` or whichever region is appropriate
+              for your speech sdk instance.
+            * You will insert your Azure speech key during build of the backend
```

Comment:
Is this true? Don't we recommend setting it as a user-secrets?

Reply:
This is just copying the recommendations listed for the Embedding and Completion blocks, will update them